client/clinet.cpp

```diff
@@ -359,7 +359,7 @@ double try_to_autoconnect(void)
     return FC_INFINITY;
   } else {
     // All errors are fatal
-    qFatal(_("Error contacting server \"%s\" at port %d "
+    qInfo(_("Error contacting server \"%s\" at port %d "
```

Comment:
Shouldn't this be at least `qWarning`? I put `qFatal` because we fail to behave as mandated by cmd line arguments, but could lower it to `qCritical` or `qWarning` if we want to continue.

Reply:
OK I used qCritical and it doesnt crash, no idea why its crashing with qFatal, at first I wanted to use printf :D

Reply:
By default, `qFatal` calls `abort()`. I could change it to call `exit(EXIT_FAILURE)` instead.